config/management/genesis/src/verify.rs

```diff
@@ -68,7 +68,7 @@ pub fn verify_genesis(
     writeln!(buffer, "Keys").unwrap();
     write_break(&mut buffer);
 
-    write_ed25519_key(&validator_storage, &mut buffer, CONSENSUS_KEY);
+    write_bls12381_key(&validator_storage, &mut buffer, CONSENSUS_KEY);
```

Comment:
Again, the CONSENSUS_KEY from `config/management/genesis/src/storage_helper.rs` was an Ed25519 one instead of a BLS one...